ingenialink/canopen/network.py

```diff
@@ -213,22 +217,23 @@ class CanopenNetwork(Network):
     PRODUCT_CODE_SUB_IX = 2
     REVISION_NUMBER_SUB_IX = 3
     NODE_GUARDING_PERIOD_S = 1
+    TRANSCEIVER_NOT_INSTALLED_ERROR_CODE = 126
```

Comment:
Why should this attribute be public?

Reply:
I just made it in accordance with the other attributes. I will make it private.